File: ledger-service/src/main/java/zachkingcade/dev/ledger/adapter/in/web/dto/account/GetAccountByIdResponse.java
package zachkingcade.dev.ledger.adapter.in.web.dto.account;

public record GetAccountByIdResponse(
        Long accountId,
        Long typeId,
        String description,
        String accountTypeName,
        String accountDisplayName,
        Long accountBalance,
        boolean active,
        String notes,
        String creditEffect,
        String debitEffect
) {
}

File: user-interface-service/src/app/adapter/ledger-service/dto/account/GETAccountByIdResponse.ts
export interface IGETAccountByIdResponse {
  accountId: number;
  typeId: number;
  description: string;
  accountTypeName: string;
  accountDisplayName: string;
  accountBalance: number;
  active: boolean;
  notes: string;
  creditEffect: string;
  debitEffect: string;
}

export type GETAccountByIdResponse = IGETAccountByIdResponse;
